### PR TITLE
pass tags to import_image

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: tests/plugins/test_import_image.py [hunk 1]
```diff
@@ -304,11 +325,28 @@ def test_import_image(tmpdir, build_process_failed, namespace, monkeypatch, reac
         (flexmock(OSBS)
          .should_receive('ensure_image_stream_tag')
          .times(DEFAULT_TAGS_AMOUNT))
-        (flexmock(OSBS)
-         .should_receive('import_image')
-         .once()
-         .with_args(TEST_IMAGESTREAM)
-         .and_return(True))
+
+        if not import_image_with_tags:
```
Comment (twaugh) [inline tests/plugins/test_import_image.py]: Small nit: when using `if not` with an `else` clause, consider omitting `not` and switching around the clauses.

Comment (rcerven) [reply tests/plugins/test_import_image.py]: changed

**Comment by lcarva** [comment]
Waiting to merge until this is merged: https://github.com/projectatomic/osbs-client/pull/776

**Comment by twaugh** [comment]
What's up with the coverage?

**Comment by rcerven** [comment]
rn updated: `post_import_image` pass tags to import_image, and only those will be imported

**Pull request merged**